projects/vega-protocol/index.js

```diff
@@ -36,10 +37,14 @@ Object.keys(config).forEach(chain => {
         fromBlock,
       })
       const blacklistedTokens = []
-      if (vega) blacklistedTokens.push(vega)
```

Comment:
sorry, this needs to be retained, we dont count count own tokens as tvl as it costs the protocol nothing to inflate tvl.